scripts/sync_model_registry_manifests.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Define root folder of Model Registry repository
+ROOT_FOLDER="${ROOT_FOLDER:-..}"
+
+# Define paths to Model Registry and Kubeflow release repositories
+MODEL_REGISTRY_REPO="${ROOT_FOLDER}/model-registry"
+KUBEFLOW_MANIFESTS_PATH="${MODEL_REGISTRY_REPO}/manifests/kustomize"
+
+# Ensure the Model Registry repository path exists
+if [ ! -d "$MODEL_REGISTRY_REPO" ]; then
+    echo "Error: Model Registry repository path does not exist."
+    exit 1
+fi
+
+# Run the script to generate OpenAPI server
+"${MODEL_REGISTRY_REPO}/gen_openapi_server.sh"
```

Comment:
Q2. not sure if you experienced you needed this command for other reasons which is not immediately clear to me? Asking because this would re-generate `.gen.go` source files, etc. but this is already taken care of and "gatekeeped" during GHA build CI jobs?